backend/src/StamAcasa.IdentityServer/appsettings.json

```diff
@@ -8,7 +8,7 @@
   },
   "AllowedHosts": "*",
   "ConnectionStrings": {
-    "ApplicationDbContextConnection": "Server=postgres;Port=5432;Database=IdentityServer;User Id=docker;Password=docker;"
+    "ApplicationDbContextConnection": "Data Source=IdentityServer.db"
```

Comment:
Any reason why you switched back to the in memory database?

Reply:
No,  I was just working locally with the in memory db